assets/css/custom.css

```diff
@@ -48,6 +48,13 @@ $accessible-gray-column-color: $gray-column-color;
   background-color: $accessible-neutral-color;
 }
 
+.accessible-colors .navbar-default .navbar-nav>li>a{
```

Comment:
The contrast can be enhanced for the non-accessible colors too: suggesting to move this to `main.css`, without `.accessible-colors` selector. The color can be more contrasted too, e.g. `#“666`.

Reply:
I agree: I thought that maybe you would not accept to change the main colors, but indeed it will be better that way!